Source/WebKit/WebProcess/glib/WebProcessGLib.cpp

```diff
@@ -259,7 +264,9 @@ void WebProcess::releaseSystemMallocMemory()
 #if PLATFORM(GTK) || PLATFORM(WPE)
 void WebProcess::setScreenProperties(const WebCore::ScreenProperties& properties)
 {
+#if PLATFORM(GTK) || (PLATFORM(WPE) && ENABLE(WPE_PLATFORM))
     WebCore::setScreenProperties(properties);
+#endif
```

Comment:
This is correct, because the guards you use match the guards around PlatformScreen.cpp.

However, it might be better to change the guards around WebProcess::setScreenProperties instead, to force higher layers of the code to understand the conditions under which the function is available, rather than to just do nothing. That's what I would do, at least. Carlos Garcia might have a preference for one way or the other.

Reply:
This is ok to me.